modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -158,6 +161,7 @@ const SaveFDSFieldsModalContent = ({
 	};
 
 	const saveFDSFields = async () => {
+		setSaveButtonDisabled(true);
```

Comment:
[nit] We have a common convention to add an empty line of code between two unrelated sibling lines of code. As an example of what is related in this file, multiple state declarations are bundled together, and multiple variable declarations. This is not a hard rule, it's not enforced by source formatter. It is kinda difficult to write a rule for this, as it is difficult to define what is 'related'. Nevertheless, we do have such set of rules for Java and JSPs, but not in JS/TS. Here, it is only a non-blocking convention.

Reply:
Thanks for caring about those little details. I usually try to keep this extra space, but without an automation is not always easy.